library/crates/BUILD.bazel

```diff
@@ -23,7 +23,25 @@ alias(
 
 alias(
     name = "cxx",
-    actual = "@raze__cxx__1_0_65//:cxx",
+    actual = "@raze__cxx__1_0_73//:cxx",
```

Comment:
This number was not a manual change: it increased unexpectedly when running `cargo raze`. It appears to occur deterministically, but given that we have pinned the version to a different number in `Cargo.toml`, I've flagged this as a bug, to be investigated soon: https://github.com/vaticle/dependencies/issues/362 .

Reply:
Does this happen on direct dependencies or indirect dependencies? Or is it both?